haystack/management/commands/update_index.py

```diff
@@ -74,21 +76,24 @@ def worker(bits):
 
 
 def do_update(backend, index, qs, start, end, total, verbosity=1):
+    start_time = now()
     # Get a clone of the QuerySet so that the cache doesn't bloat up
     # in memory. Useful when reindexing large amounts of data.
     small_cache_qs = qs.all()
     current_qs = small_cache_qs[start:end]
 
     index.pre_process_data(current_qs)
 
+    # FIXME: Get the right backend.
+    backend.update(index, current_qs)
+
+    time_delta = now() - start_time
+
     if verbosity >= 2:
         if hasattr(os, 'getppid') and os.getpid() == os.getppid():
-            print("  indexed %s - %d of %d." % (start + 1, end, total))
+            logger.info("  indexed {} - {} of {} in {}.  Took {}".format(start + 1, end, total, time_delta))
```

Comment:
To your first comment, we probably don't need them in the logs, but the command still prints to stdout, and the indentation is nice for that purpose, but I can remove if it makes the logs unreadable.

Good catch on the placeholder mismatch.  I was surprised that I hadn't caught that when I tested it.  Looking at it now, I don't see how that branch of code would ever get executed.  @ruebrenda can you think of any circumstance where a process's id would match it's parent process id?

I'll fix the number of place holders in the meantime, but if that condition makes as little sense as i think it does, I'll remove it entirely.